Tasks/KubernetesV1/src/kubernetes.ts

```diff
@@ -107,11 +107,12 @@ function executeKubectlCommand(clusterConnection: ClusterConnection, command: st
     return commandImplementation.run(clusterConnection, command, (data) => result.push(data))
         .fin(function cleanup() {
             console.log("commandOutput" + result);
-            var commandOutputLength = result.length;
+            const resultString = result.toString();
```

Comment:
Can we move this before console.log statement..

Reply:
console.log will handle js object.